docs/multicast-guide.md

```diff
@@ -0,0 +1,153 @@
+GROUP       PODS
+225.1.2.3   default/mcjoin, namespace/pod
+224.5.6.4   default/mcjoin
+```
+
+### Inbound and outbound multicast traffic statistics
+
+Please refer to the corresponding [antctl page](antctl.md#multicast-commands).
+
+### Multicast NetworkPolicy statistics
+
+[Antrea NetworkPolicystats](feature-gates.md#networkpolicystats) also apply to multicast NetworkPolicy.
+
+## Use case example
+
+In this section, we will show how multicast video steaming works in
+Antrea using [VLC](https://www.videolan.org/vlc/).
+
+To start a sample video streaming server, we create a Pod streaming sample video
+to the multicast IP address `239.255.12.42` with TTL 6.
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-sender -- vlc --intf ncurses --vout dummy --aout dummy sample-video.mp4 --sout udp:239.255.12.42 --ttl 6 --repeat
+```
+
+It's expected to see outbound multicast traffic from this Pod
+by running `antctl get podmulticaststats` from Antrea Agent local to this Pod,
+which means the Pod is sending video streams outside the Pod.
+
+You will also find a multicast route, which indicates traffic from Antrea gateway
+to the transport interface by running `ip mroute`.
+
+```bash
+ip mroute
+(pod-ip, 239.255.12.42)     Iif: antrea-gw0 Oifs: transport interfaces State: resolved
+```
+
+A sample client is created as well with the following command:
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-receiver -- vlc --intf ncurses --vout dummy --aout dummy udp://@239.255.12.42 --repeat
+```
+
+It's expected to see outbound multicast traffic from this Pod by running
+`antctl get podmulticaststats` from the Antrea Agent local to this Pod,
+which means the Pod is able to receive the video stream.
+
+Also, kubectl command `kubectl get multicastgroups` will print that `vlc-receiver`
+has joined multicast group `239.255.12.42`.
+
+## Limitations
+
+This feature is currently only supported for IPv4 Linux clusters.
+Support for Windows and IPv6 will be added in the future.
+
+### Traffic in local network control block
+
+Multicast IP that in [Local Network Control Block](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xhtml#multicast-addresses-1) (224.0.0.0/24)
+should work only in encap mode. Multicast traffic destined for that IPs
+are not expected to be forward, therefore, no multicast route will be configured,
+and external hosts are not working either.
+
+### Linux kernel
+
+You may find some traffic is not routed if the Node environment meets the following specifications:
+
+1. Node kernel version under 5.4
+2. Node network doesn't support IGMP snooping
+3. Lots of multicast traffic with group IPs that won't be received by the Node
+   (no Pod has not joined that group in that Node)
+
+### Encap mode
+
+In encap mode, multicast packets sent from Pod cannot be forwarded to more than one
+network interface on the host. Therefore Pods to external hosts multicast is not supported
```

Comment:
interfaces

on the Node

Reply:
We should talk about the issue first. So, move "Multicast traffic from Pods to external hosts is not supported in encap mode" to be the first sentence.

I feel we can remove the reason sentence, but if you want to keep it" "This is because ..."

Reply:
updated. Reason sentence removed.